shoop/core/management/commands/shoop_extractmessages.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
```

Comment:
I'm not sure why the management command is necessary in the first place. [The script in `_misc` is documented, after all...](https://github.com/shoopio/shoop/blob/master/doc/dev_tools.rst#i18n-catalog-extraction)